src/lib/connectors/shared/remote-debugging-connector.ts

```diff
@@ -291,6 +284,25 @@ export class Connector implements IConnector {
         return response;
     }
 
+    private isFaviconLink(element: IAsyncHTMLElement) {
```

Comment:
@sarvaje Maybe make this a more general thing (e.g.: `hasAttributeWithValue(element, value)`) , and move it into `utils/misc`?